src/gwt/src/org/rstudio/studio/client/application/Application.java

```diff
@@ -426,6 +426,12 @@ public void onError(ServerError error)
       });
    }
 
+   @Handler
+   void onShowSplashScreen()
+   {
+      Desktop.getFrame().showSplashScreen();
```

Comment:
Does this need to do anything for RStudio Server? Or should we hide / disable the command there?

Reply:
Oops! I see exactly that is happening just below.